samples/azure-quantum/shorter-quantum-circuits/shorter-quantum-circuits-dataset.ipynb

```diff
@@ -0,0 +1,782 @@
+{
```

Comment:
Some global comments on the notebook:
* Consider to publish the notebook with already pre-evaluated code cells
* Consider moving the imports into a top-level code cell instead of repeating them for the code cells individually

Reply:
For the first point: added notebook with pre-evaluated cells. Was not sure what is convention here. 
For the second point: it is intentional so people can easily copy past parts they are interested in into their code.

Reply:
I think for the first point is case by case basis. Here I think it makes sense to have it as a read-only mode as well, so pre-evaluated cells are nice to have.